packages/calypso-products/src/plans-list.tsx

```diff
@@ -1574,6 +1576,25 @@ PLANS_LIST[ PLAN_P2_FREE ] = {
 };
 
 // Brand new WPCOM plans
+PLANS_LIST[ PLAN_WPCOM_STARTER ] = {
+	...getDotcomPlanDetails(),
+	group: GROUP_WPCOM,
+	type: TYPE_STARTER,
+	term: TERM_ANNUALLY,
+	getTitle: () => i18n.translate( 'WordPress Starter' ),
+	getProductId: () => 1033,
+	getStoreSlug: () => PLAN_WPCOM_STARTER,
+	getPathSlug: () => 'starter-plan',
```

Comment:
Drive-by comment -- this should just be `starter`, actually (that's how we want it to work elsewhere)?

Reply:
Thanks, will update. Assuming this is safe right? I'm not too familiar with most of these things. I think I was trying to connect the dots with your comment here 711-gh-Automattic/martech#issuecomment-1114502508